app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,46 @@
+require 'uri'
+require 'net/http'
+require 'json'
+
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(benchmark_run_id)
+
+    benchmark_run = BenchmarkRun.find_by_id(benchmark_run_id)
+
+    category = benchmark_run.result.keys[0]
+
+    previous_benchmark_runs = BenchmarkRun.where('created_at < ?', benchmark_run.created_at).where(initiator_type: Commit, benchmark_type: benchmark_run.benchmark_type,
+                     benchmark_result_type: benchmark_run.benchmark_result_type).limit(1000)
+
+    commit_results = previous_benchmark_runs.map { |run| run.result[category].to_f }
+
+    results_param = average(commit_results) + 2*standard_deviation(commit_results)
+
+    if benchmark_run.result[category].to_f > results_param
+      create_issue
+    end
+  end
+
+  def create_issue
+    uri = URI.parse("https://api.github.com/repos/shahsaurabh0605/docker-practice/issues")
+    http_client = Net::HTTP.new(uri.host, uri.port)
+    http_client.use_ssl = true
+    request = Net::HTTP::Post.new(uri.path)
+    request.basic_auth 'RubyBenchBot', Rails.application.secrets.github_password
+    request.body = {'title' => 'Benchmark Regression Detected'}.to_json
```

Comment:
You need to include more details in the issue in order for the dev to quickly identify which benchmark has regressed